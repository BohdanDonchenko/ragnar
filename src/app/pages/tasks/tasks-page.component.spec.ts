import { TestBed, async } from '@angular/core/testing';

import { TasksPageComponent } from './tasks-page.component';

describe('TasksPageComponent', () => {
  describe('Snapshot', () => {
    beforeEach(async(() => {
      TestBed.configureTestingModule({
        declarations: [TasksPageComponent],
      }).compileComponents();
    }));

    test('check snapshot', () => {
      const fixture = TestBed.createComponent(TasksPageComponent);

      expect(fixture).toMatchSnapshot();
    });
  });
});
