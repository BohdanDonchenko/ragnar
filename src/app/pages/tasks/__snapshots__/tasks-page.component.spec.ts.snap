// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TasksPageComponent Snapshot check snapshot 1`] = `
<tasks-page>
  <h2>
    Tasks component
  </h2><div
    class="tasks-component"
  >
     Nothing to show. Just an example of routing.
  
  </div>
</tasks-page>
`;
